scripts/generate-patch

```diff
@@ -37,6 +37,9 @@ for f in packages/*; do
   if [[ -f ${f}/package.yaml ]]; then
   	if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
 			url=$(cat ${f}/package.yaml | yq r - url)
+			if [[ -z ${url} ]]; then
```

Comment:
continue works here because we do not want to generate a patch at all for charts that do not have a url

Reply:
Sounds good to me, this is linked to https://github.com/rancher/charts/pull/607#discussion_r483290617 so I'll resolve both conversations.